app-crypt/p11-kit/p11_kit-0.23.9.recipe

```diff
@@ -0,0 +1,93 @@
+SUMMARY="Provides a way to load and enumerate PKCS#11 modules"
+DESCRIPTION="p11-kit provides a way to load and enumerate PKCS#11 modules. \
+Provides a standard configuration setup for installing PKCS#11 modules \
+in such a way that they're discoverable."
+HOMEPAGE="https://p11-glue.freedesktop.org/p11-kit.html"
+COPYRIGHT="2017 Red Hat, Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/p11-glue/p11-kit/releases/download/$portVersion/p11-kit-$portVersion.tar.gz"
+CHECKSUM_SHA256="e1c1649c335107a8d33cf3762eb7f57b2d0681f0c7d8353627293a58d6b4db63"
+SOURCE_DIR="p11-kit-$portVersion"
+PATCHES="p11_kit-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	p11_kit$secondaryArchSuffix = $portVersion
+	cmd:p11_kit$secondaryArchSuffix = $portVersion
+	cmd:trust$secondaryArchSuffix = $portVersion
+	lib:libp11_kit$secondaryArchSuffix = $portVersion
+	lib:p11_kit_proxy$secondaryArchSuffix = $portVersion
```

Comment:
```diff
-ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
-SECONDARY_ARCHITECTURES="x86"
+SECONDARY_ARCHITECTURES="?x86"

+libVersion="0.3.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	p11_kit$secondaryArchSuffix = $portVersion
 	cmd:p11_kit$secondaryArchSuffix = $portVersion
 	cmd:trust$secondaryArchSuffix = $portVersion
-	lib:libp11_kit$secondaryArchSuffix = $portVersion
+	lib:libp11_kit$secondaryArchSuffix = $libVersionCompat
-	lib:p11_kit_proxy$secondaryArchSuffix = $portVersion
+	lib:p11_kit_proxy$secondaryArchSuffix = $libVersionCompat
	"
```

Reply:
What is libVersion? If it means "the version of the library" then it's 0.23.9, right?

Reply:
Alright I was wrong.